setup.py

```diff
@@ -31,11 +31,11 @@
             'html = aleph.ingest.html:HtmlIngestor',
             'djvu = aleph.ingest.djvu:DjVuIngestor',
             'img = aleph.ingest.image:ImageIngestor',
-            'email = aleph.ingest.email:EmailFileIngestor',
-            'mbox = aleph.ingest.email:MboxFileIngestor',
+            'email = aleph.ingest.email_:EmailFileIngestor',
```

Comment:
Oh. My. God.

But why does this pull request need to fix this? Was it never working?